app/events/applications/application_deleted_event.rb

```diff
@@ -4,12 +4,16 @@ class Applications::ApplicationDeletedEvent < ApplicationRelatedEvent
 
   # @param [Cinstance] application
   def self.create(application)
+    service_id = application.service_id
     new(
       application: MissingModel::MissingApplication.new(id: application.id),
+      service_id: service_id,
+      user_key: application.user_key,
```

Comment:
https://github.com/3scale/porta/pull/1815#discussion_r412804008

![image](https://user-images.githubusercontent.com/11318903/80162801-92c27580-85d4-11ea-94cc-68a66f83a1c2.png)

Reply:
Waaaait, i think that we may not need this anymore 🤔 but i will check more in-depth

Reply:
RIght, we do not need it. The application is removed using the ID, hence it does not need any other extra removal.